### DISCUSSION TITLE
About .github/workflows/CloseInactiveIssues.yml

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install-GUI#58] [Q&A] [answered]
In your Repositories forked by me/others there's now this GitHub file.. ..and the changes in that file in our forks now says:
> This branch is X commits behind Romanitho:main

How can we disable the check of this file (or do we need to Fetch the upstream when it changes)?

**Comment by Romanitho** [comment, marked as answer]
I don't know at all. But it should not change anymore :)
